app/scripts/produzenten/detail/produzentendetail.controller.js

```diff
@@ -45,9 +53,20 @@ angular.module('openolitor-admin')
       };
 
       $scope.save = function() {
-        return $scope.produzent.$save(function() {
-          $scope.produzentForm.$setPristine();
-        });
+        var listOfNicknames = lodash.map($scope.produzenten, 'kurzzeichen');
+        //new producer
+        if (isNew && lodash.filter(listOfNicknames, function(n){ return n === $scope.produzent.kurzzeichen}).length > 0){
+            alertService.addAlert('error', gettext('Diese produzent existiert bereits'));
```

Comment:
Nouns are always uppercase in german and since Produzent is masculin, it's: 'Dieser Produzent existiert bereits'

Reply:
done